src/test/java/teammates/test/cases/browsertests/FeedbackContributionQuestionUiTest.java

```diff
@@ -180,7 +180,7 @@ private void testCopyOptions() {
         feedbackEditPage.clickNewQuestionButton();
         feedbackEditPage.selectNewQuestionType("CONTRIB");
         feedbackEditPage.fillQuestionTextBoxForNewQuestion("q2, contribution qn");
-        assertEquals("Please select a visibility option",
```

Comment:
Sorry, I meant the visibility hint. This is what I get when I create an new essay question, and this seems the same for any new non-contrib question -

![capture](https://user-images.githubusercontent.com/11662001/27004838-6e2b2892-4e2e-11e7-8448-17c4561f0bfb.JPG)

So should I change the code so that the above hint is displayed along with `Please select a visibility option` on the button?